exporter/logger/googlecloudexporter/client_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright  The OpenTelemetry Authors
```

Comment:
Here and elsewhere, can you assign copyright to Google?  That will allow us to use it in the implementation:

```
// Copyright 2022 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
```

Reply:
I still see 11 more that need to be updated